source/Ship.h

```diff
@@ -615,6 +615,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Various energy levels:
 	double shields = 0.;
 	double hull = 0.;
+	double shieldsWear = 0.;
+	double hullWear = 0.;
```

Comment:
I get the impression that those values also need to be saved to savegames (for ships that are not at a spaceport when a game is saved.)